spec/lib/open_project/text_formatting_spec.rb

```diff
@@ -574,8 +574,14 @@ def controller
     subject(:html) { format_text(wiki_text) }
 
     context 'w/ request present' do
-      let(:request) { ActionController::TestRequest.new }
-      let(:url_for) { '/test' }
+      let(:request) {
+        ActionController::TestRequest.new(
+          Rack::MockRequest.env_for('/test',
+            'HTTP_HOST'       => 'test.host',
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Unnecessary spacing detected.